sdv/model.py

```diff
@@ -78,119 +74,10 @@ def set_context(self, ctx: List[str]):
         context.set(ctx)
 
 
-class Model(Node):
```

Comment:
I'm not sure why the base models are moved to the bottom ?

Reply:
Moved because "Model" depends on "BatchSetBuilder" depends on DataPoint classes

Reply:
As discussed, this can be improved when updating the flaunt queries.